src/settings.ts

```diff
@@ -63,6 +64,20 @@ export class TrelloSettings extends PluginSettingTab {
       });
   }
 
+  private openInDesktopSetting(containerEl: HTMLElement, settings: PluginSettings): void {
+    if (Platform.isMacOS || Platform.isWin) {
```

Comment:
I'm fine with hiding this setting on other platforms, as long as we also override the saved settings on other platforms. I think a check in `plugin.ts` after loading settings, but before initializing the state:

```ts
    // Set up data and default data.
    const savedData: PluginData | undefined = await this.loadData();
    const data: PluginData = Object.assign({}, DEFAULT_DATA, savedData);
    data.settings = Object.assign({}, DEFAULT_SETTINGS, savedData?.settings);
    data.settings.customUi = Object.assign({}, DEFAULT_SETTINGS.customUi, savedData?.settings.customUi);

    // Add something like this
    if (!Platform.isWin && !Platform.isMacOS) {
        data.settings.openInDesktop = false;
    }

    this.state = new PluginState(this, data);
```

It's not a big deal but I know I use a synced obsidian vault on multiple platforms. I wouldn't want someone to enable it on a Mac, then open Obsidian on Linux and have everything be broken.

Reply:
@nathonius OK, I will need to test this, and it's a busy week, leave it with me